addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/generators/PathGenerator.java

```diff
@@ -38,11 +38,10 @@ public String generateFullPath(OperationModel operationModel) {
                     continue;
                 }
                 String parameterType = parameter.getIn();
+                String value = dataGenerator.generate(parameter.getName(), parameter);
```

Comment:
I'd suggest reverting this, it avoids generating a value when not needed (header parameters).